src/main/java/bleuauction/bleuauction_be/server/notice/service/NoticeService.java

```diff
@@ -37,25 +63,49 @@ public List<Notice> findNoticesByStatus(NoticeStatus status) {
 
   @Transactional(readOnly = true)
   public Notice findOne(Long noticeNo) {
-    return noticeRepository.findOne(noticeNo);
+    return noticeRepository.findByNoticeNo(noticeNo);
   }
 
 
   //노티스 삭제(N)
   @Transactional
-  public void deleteNotice(Long noticeNo) {
-    Notice notice = noticeRepository.findOne(noticeNo);
-    notice.delete();
+  public void deleteNotice(Long noticeNo, Member member) {
+    if (member.getMemberCategory()==MemberCategory.A) {
+      Notice notice = noticeRepository.findByNoticeNo(noticeNo);
+      notice.delete();
+      if (notice.getNoticeAttaches() != null && !notice.getNoticeAttaches().isEmpty()) {
+        for (Attach attach : notice.getNoticeAttaches()) {
+          attachService.changeFileStatusToDeleteByFileNo(attach.getFileNo());
+        }
+      }
+    }
   }
```

Comment:
요기도 요론식으로, If문을 제거해볼 수 있을 것 같아요.

```suggestion
    @Transactional
    public void deleteNotice(Long noticeNo, Member member) {
        if(!MemberCategory.A.equals(member.getMemberCategory())) {
            // 관리자가 아니기 떄문에, 예외처리 
        }
        
        Notice notice = noticeRepository.findByNoticeNo(noticeNo);
        notice.delete();
        if (notice.getNoticeAttaches() != null && !notice.getNoticeAttaches().isEmpty()) {
            for (Attach attach : notice.getNoticeAttaches()) {
                attachService.changeFileStatusToDeleteByFileNo(attach.getFileNo());
            }
        }
    }
```